sdk.conf

```
@@ -1 +1 @@
-osgeo4a_version=20200611
+osgeo4a_version=gdal
```

```suggestion
osgeo4a_version=gdal

```

Create a fake commit please